sv_pipeline/exome/load_data.py

```diff
@@ -439,6 +449,7 @@ def main():
     p.add_argument('--skip-sample-subset', action='store_true')
     p.add_argument('--write-subsetted-bed', action='store_true')
     p.add_argument('--ignore-missing-samples', action='store_true')
+    p.add_argument('--is-merging-sample-call', action='store_true')
```

Comment:
per my comment on the ticket, we need to have the default behavior change and we want the `--is-new-joint-call` flag to be something you have to set explicitly